RecAlgo/test2.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+# In[1]:
+
+
+from models import TFIDFBookRec
+from utils import Book
+import pandas as pd
+import numpy as np
+
+
+# In[2]:
+
+
+'''
+test_book: a Book instance, contains six attributes: [ISBN, title, author, year, publisher, imageUrl].
```

Comment:
whats this file about?

Reply:
oh oops i shouldn't have committed this my bad, i converted the test notebook to a .py file so it was easier to run